rocAL/include/pipeline/commons.h

```diff
@@ -111,6 +118,7 @@ enum class RocalROIType {
     LTRB = 0,
     XYWH
 };
+REGISTER_ENUM(RocalROIType)
```

Comment:
Non-blocking: Associated enum is defined as . Consider explicitly specifying the underlying type (e.g., ) to keep ABI stable and avoid surprises across compilers.